src/utils/config.ts

```diff
@@ -96,15 +97,15 @@ export async function findTargetInConfiguration(
   if (targetName) {
     try {
       target = await getLocalTarget(targetName)
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
`e` shouldn't be of type `any`, because it has to be an object with property `message`.
The same applies for the following `catch` blocks

Reply:
We cannot define type of error in catch, it has to be `any` or `unknown`